src/main/app/claims/claimModelConverter.ts

```diff
@@ -0,0 +1,64 @@
+import DraftLegalClaim from 'drafts/models/draftLegalClaim'
+import Defendant from 'drafts/models/defendant'
+import { OtherDamages } from 'forms/models/otherDamages'
+
+export class ClaimModelConverter {
+
+  static convert (draftClaim: DraftLegalClaim): object {
+    this.convertClaimantDetails(draftClaim)
+    this.convertDefendantDetails(draftClaim)
+
+    draftClaim['reason'] = draftClaim.summary.text
+
+    if (draftClaim.personalInjury.generalDamages) {
+      draftClaim.personalInjury.generalDamages = draftClaim.personalInjury.generalDamages.dataStoreValue as any
+      delete draftClaim.personalInjury.personalInjury
+    }
+
+    if (draftClaim.housingDisrepair.generalDamages) {
+      draftClaim.housingDisrepair['costOfRepairsDamages'] = draftClaim.housingDisrepair.generalDamages.dataStoreValue
+      delete draftClaim.housingDisrepair.generalDamages
+    }
+
+    draftClaim.housingDisrepair.otherDamages = draftClaim.housingDisrepair.otherDamages.dataStoreValue as any
+    if (draftClaim.housingDisrepair.otherDamages as any === OtherDamages.NONE.dataStoreValue) {
```

Comment:
yeah I had to, it wouldn't compile because otherDamages has a different type on the object passed in